Assets/_BForBoss/_Entities/Scripts/LifeCycleBehaviour.cs

```diff
@@ -1,3 +1,4 @@
+using System;
```

Comment:
wait is this needed?

Reply:
No good catch, I'd originally made changes here and forgot to remove the usings when I undid them